Mu2eG4/geom/bfgeom_reco_v01.txt

```diff
@@ -14,7 +14,8 @@
 #include "Mu2eG4/geom/bfgeom_v01.txt"
 
 vector<string> bfield.innerMaps = {
-  "BFieldMaps/Mau13/DSMap.header"
+  "BFieldMaps/Mau13/DSMap.header",
+  "BFieldMaps/Mau13/DSExtension.header"
```

Comment:
I am confused about the use case.  CRV digitization and reconstruction don't access the BField map, so why does this matter here?  if the use case is pileup simulation, that job uses  geom_common_no_tsu_ps.txt   which includes DSExtension.

Reply:
You're right.  I will back this one out.  I will also remove these two lines from geom_common_no_tsu_ps.txt

  "BFieldMaps/Mau13/PStoDumpAreaMap.header",
  "BFieldMaps/Mau13/ProtonDumpAreaMap.header",